### ISSUE TITLE
Logging request payload

### ISSUE THREAD

**Issue by shinzui** [pinojs/hapi-pino#13]
Is it possible to add an option to log the request's payload?

**Comment by mcollina** [comment]
This is currently not possible because of https://github.com/pinojs/hapi-pino/blob/master/index.js#L15-L18.

But I'd be very glad to receive a PR to add support for custom serializers.

**Comment by juanjoLenero** [comment]
Wouldn't this be a matter of not overwriting the specific serializers?
E.g. Going from this:
```js
  options.serializers = options.serializers || {}
  options.serializers.req = asReqValue
  options.serializers.res = pino.stdSerializers.res
  options.serializers.err = pino.stdSerializers.err
```
To this:
```js
  options.serializers = options.serializers || {}
  options.serializers.req = options.serializers.req || asReqValue
  options.serializers.res = options.serializers.res || pino.stdSerializers.res
  options.serializers.err = options.serializers.err ||pino.stdSerializers.err
```

**Comment by mcollina** [comment]
yeah! would you like to send a PR? We might also think about embedding http://npm.im/pino-noir

**Comment by felixheck** [comment]
@shinzui @juanjoLenero @mcollina any progress on this task?

**Comment by mcollina** [comment]
@felixheck would you like to send a PR?

**Comment by felixheck** [comment]
It's just the change mentioned by @juanjoLenero?
Or wouldn't it be easier with

```
options.serializers.req = asReqValue.bind(options.logReqPayload)

...

function asReqValue (addPayload, req) {
  const raw = req.raw.req

  return {
    id: req.id,
    method: raw.method,
    url: raw.url,
    headers: raw.headers,
    remoteAddress: raw.connection.remoteAddress,
    remotePort: raw.connection.remotePort,
    payload: addPayLoad ? req.payload : undefined
  }
}
```

**Comment by mcollina** [comment]
I think we need the changed mentioned in https://github.com/pinojs/hapi-pino/issues/13#issuecomment-303475554, and yours is good as well.

**Comment by felixheck** [comment]
Okay, so I'm gonna combine both and send a PR.

**Comment by mcollina** [comment]
#18 has been released as 1.7.0.
